src/porepy/viz/exporter.py

```diff
@@ -366,21 +366,37 @@ def _export_g(self, data: Dict[str, np.ndarray], time_step):
     # ------------------------------------------------------------------------------#
 
     @pp.time_logger(sections=module_sections)
-    def _export_gb(self, data: List[str], time_step: float):
+    def _export_gb(self, data: List[str], time_step: float) -> None:
+        """Export the entire GridBucket and additional data to vtu.
+
+        Parameters:
+            data (List[str]): Data to be exported in addition to default GridBucket data.
+            time_step (float) : Time step, to be appended at the vtu output file.
+        """
         # Convert data to list, or provide an empty list
         if data is not None:
             data = np.atleast_1d(data).tolist()
         else:
             data = list()
 
-        fields = Fields()
-        if len(data) > 0:
-            fields.extend([Field(d) for d in data])
+        # Extract data which is contained in nodes (and not edges).
+        # To make the unit test 'test_vtk' successfull, keep the sorting from data.
+        node_data = list()
+        for key in data:
+            for _, d in self.gb.nodes():
+                if pp.STATE in d and key in d[pp.STATE]:
+                    node_data.append(key)
+                    continue
```

Comment:
The rational here is that you have found this key, added it to `node_data`, and don't want to add it once more? If so, should it not be `break `rather than `continue`? 

Anyhow, please document the functionality, this is the future me may wonder about.